geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegionQueue.java

```diff
@@ -668,4 +679,27 @@ public boolean isReadyForPeek() {
         && !this.eventSeqNumDeque.isEmpty() && getBucketAdvisor().isPrimary();
   }
 
+  List<Object> getQueueList() {
```

Comment:
This method should probably be marked as `@VisibleForTesting` and maybe renamed to make it clear that it's a test helped method.